x-pack/plugins/fleet/scripts/create_agents/create_agents.ts

```diff
@@ -25,6 +25,8 @@ const printUsage = () =>
     [--kibana]: full url of kibana instance to create agents and policy in e.g http://localhost:5601/mybase, defaults to http://localhost:5601
```

Comment:
this is just changes to the test script to make creating envs with lots of agent policies easier.